libraries/3rdparty/CMakeLists.txt

```diff
@@ -1,6 +1,20 @@
 # -------------------------------------------------------------------------------------------------
 # 3rdparty libraries
 # -------------------------------------------------------------------------------------------------
+# http-parser - HTTP message parser
+if(EXISTS "${AFR_3RDPARTY_DIR}/jsmn")
```

Comment:
```jsmn``` is not the right thing to check here. This should be ```http-parser```

Reply:
Thanks!